packages/form/addon/components/cf-field/input/radio.hbs

```diff
@@ -22,4 +22,10 @@
       {{option.label}}
     {{/if}}
   </label>
-{{/each}}
\ No newline at end of file
+{{/each}}
+{{#if (and @field.optional @field.answer.value)}}
+  <br />
```

Comment:
Can we replace this with `uk-margin-small-top` on the link?

Reply:
Sure. I'll give it a shot and push. brb

Reply:
`a` is an inline element. So we would have to add `uk-display-block` at least. Extra margin to that does not look good (to much of a gap between options and reset) *imho*. 

That's how it looks with `uk-display-block` only:
![image](https://github.com/projectcaluma/ember-caluma/assets/10029904/f0ef04c8-2798-4635-a4bb-9c12b369c98e)

And that's with the extra `uk-margin-small-top`:
![image](https://github.com/projectcaluma/ember-caluma/assets/10029904/fe649b66-eaaf-48ec-b5ec-57daffdd926f)

Reply:
Let's keep it like it is then. Thanks for investigating.

Reply:
Well, it doesn't look to bad paired with follow up questions. And it has the same margin then like *hints*. 
![image](https://github.com/projectcaluma/ember-caluma/assets/10029904/2eee2258-b160-4d1d-932c-5204c60ca4ba)

Will add the small margin. ;)

Reply:
Ok to late :D